java/ql/src/experimental/Security/CWE/CWE-665/InsecureRmiJmxEnvironmentConfiguration.qhelp

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>An improperly set environment variable during the creation of an RMI or JMX server can lead 
```

Comment:
I find this confusing since it seems like this vulnerability is affecting any RMI services and my understanding, please correct me if Im wrong, is that it only affects RMI when used with JMX. So for example `RMIConnectorServer` [docs](https://docs.oracle.com/javase/10/docs/api/javax/management/remote/rmi/RMIConnectorServer.html) reads:

> A JMX API connector server that creates RMI-based connections from remote clients. Usually, such connector servers are made using JMXConnectorServerFactory. However, specialized applications can use this class directly, for example with an RMIServerImpl object.

Should we reword the description and talk only about JMX to avoid this confusion?

Reply:
Yeah this is definitely described incorrectly. This vulnerability should only occur when JMX is in use, not with RMI in general.